docs/editing_guidelines.md

```diff
@@ -90,6 +90,25 @@ If the main source for an assertion is a term in another ontology, the short ide
 
 ORCID identifiers may also be used when the only available source for an assertion is an individual researcher. This should be done sparingly.
 
+**Technical details of adding a cross-reference using Protégé**:
+
+For all: In the "Create Annotation" window, select the annotation property **database_cross_reference**.
+
+If the cross-reference can be represented as a CURIE and is not an ORCID: Enter the CURIE, using the bioregistry standard prefix, as a Value on the "Literal" tab. Leave Datatype empty.
+
+Non-CURIE values (e.g., URLs) are discouraged, but when used are also entered the same way, i.e., as a literal string with no Datatype selected.
+
+In cases where more than one CURIE is available for a resource, e.g., both a PMID and a doi are available, there is no preference for one over the other.
```

Comment:
```suggestion
In cases where more than one CURIE is available for a resource, e.g., both a PMID and a doi are available, we recommend using PMID by default, as it is a more _semantically meaningful_ identifier: you can immediately see that the cross-reference is to a paper, while a doi could be essentially anything. However, both doi and PMID are acceptable.
```

@bvarner-ebi this is my personal preference, it is a bit more "guiding" them simply saying "it does not matter". What do you think?

Reply:
Thanks for the suggestion, @matentzn. How about:

In cases where more than one CURIE is available for a resource, either is acceptable, but using the more semantically specific identifier is recommended. For example, when both a PMID and a doi are available for a resource, using the PMID is recommended since it indicates the cross-reference points to a paper, as opposed to a doi which could point to any digital object.